lib/moon_web/components/page_section.ex

```diff
@@ -28,4 +30,8 @@ defmodule MoonWeb.Components.PageSection do
     </section>
     """
   end
+
+  defp get_as_anchor_id(title) do
```

Comment:
Please do not use defp. Always extract defp function to be reusable to some other module.

Also get_as_anchor_id() is defined in MoonWeb.Components.PreviewCodeButton